src/components/ProjectDashboard/components/NftRewardsPanel/hooks/nftRewards.ts

```diff
@@ -0,0 +1,100 @@
+import { PayProjectFormContext } from 'components/Project/PayProjectForm/payProjectFormContext'
```

Comment:
rename this hook file to `useNftRewards.ts`

Reply:
Done